<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="USMUMAANNALDAS2" name="DataProviderTest" tests="2" failures="2" timestamp="29 Dec 2016 08:30:03 GMT" time="0.613" errors="0">
  <testcase name="testLogin" time="0.01" classname="multipleDataProviders.Login">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at multipleDataProviders.Login.testLogin(Login.java:30)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase name="testLogin" time="0.0" classname="multipleDataProviders.Login">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at multipleDataProviders.Login.testLogin(Login.java:30)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
</testsuite> <!-- DataProviderTest -->
